virttest/qemu_monitor.py

```diff
@@ -2010,6 +2010,12 @@ def query(self, what, debug=True):
         """
         return self.info(what, debug)
 
+    def query_cpus_fast(self, debug=True):
+        """
+        Return the list of vcpus infomation
```

Comment:
Typo, information.

Reply:
Updated, and changed to let qemu_vm invoke qemu_monitor directly.